cmd/api/main.go

```diff
@@ -48,22 +49,51 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+type ConfigFile struct {
+	DB_USER     string `mapstructure:"DB_USER"`
+	DB_PASSWORD string `mapstructure:"DB_PASSWORD"`
```

Comment:
Do we want to allow DB credentials to be stored in plaintext? Will Viper also allow these to be set with env vars, and the other settings to be populated via the file?

Reply:
it's optional to have the password in the config file, if a config item is set as a env var, viper will use that over the config file version [1]. 

[1](https://github.com/spf13/viper/blob/master/viper.go#L1399:L1400)